sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -87,8 +87,8 @@
     @Parameter(names = "-numberOfCollectionForCtl", description = "Number of collections for ctl load")
     private int numberOfCollectionForCtl = 4;
 
-    @Parameter(names = "-readWriteQueryPct", description = "Comma separated read write query workload percent")
-    private String readWriteQueryPct = "90,9,1";
+    @Parameter(names = "-readWriteQueryReadManyPct", description = "Comma separated read write query readMany workload percent")
+    private String readWriteQueryReadManyPct = "90,8,1,1";
```

Comment:
You need to make changes in docker run scripts otherwise ctl workload will fail https://github.com/Azure/azure-sdk-for-java/blob/4f2e85d8827abd87d1a803867e0c49fbf8e5db56/sdk/cosmos/azure-cosmos-benchmark/ctl/run_benchmark.sh#L30

Reply:
Done